fw/t/unit/test_http_parser.c

```diff
@@ -47,12 +47,19 @@
 #include "msg.c"
 #include "http_msg.c"
 
-static const int PRIMES[] = { 1, 2, 3, 5, 7, 11, 13, 17 };
+/* Compile the test itself w/o autovectorization. */
+#pragma GCC reset_options
```

Comment:
With #1520 merged we don't need GCC pragmas any more

Reply:
Done